### ISSUE TITLE
WARNING: Reference names in file with genomic features of type "mRNA" do not match any chromosome.

### ISSUE THREAD

**Issue by sammyjava** [ablab/quast#69]
I don't understand this warning: I'm using the official GRCh38 FASTA and GFF, and the mRNA records in the GFF have the same chromosome names as the FASTA. Here's my command line:
```
/home/shokin/quast-5.0.0/quast.py -r /home/shokin/share/GRCh38_top_level.fa -g mRNA:/home/shokin/share/ref_GRCh38.p12_top_level.gff3 \
	-m 100 -t 16 -L -e -i 65 --min-identity 95.0 \
	-o SRR1609370-trinity/quast_results SRR1609370-trinity/Trinity-GG.fasta
```
And here's the warning I get:
```
2018-09-18 10:46:22
Running Genome analyzer...
  NOTICE: No file with operons were provided. Use the -O option if you want to specify it.
  Loaded 119289 genomic features of type "mRNA"
  WARNING: Reference names in file with genomic features of type "mRNA" do not match any chromosome. Check your genomic feature file(s).
  SRR1609370-trinity
```
Here's a sample mRNA line from the GFF:
```
NC_000001.11    BestRefSeq      mRNA    450740  451678  .       -       .       ID=rna41;Parent=gene23;Dbxref=GeneID:729759,Genbank:NM_001005221.2,HGNC:HGNC:31275;Name=NM_001005221.2;gbkey=mRNA;gene=OR4F29;product=olfactory receptor family 4 subfamily F member 29;transcript_id=NM_001005221.2
```
and here's the corresponding chromosome header in the FASTA:
```
>NC_000001.11 Homo sapiens chromosome 1, GRCh38 Primary Assembly
```
Sure looks like the same chromosome name to me!

**Comment by alexeigurevich** [comment]
Hi,
For matching genes, we require exact matches of reference names in FASTA and GFF and in your particular case:
`NC_000001.11` is a substring but not the exact match to `NC_000001.11 Homo sapiens chromosome 1, GRCh38 Primary Assembly`.
In some previous QUAST version we allow substring matches but this caused problems/incorrect annotations in cases like `chr1` and `chr11`. 

Anyway, I see the problem and we could probably solve it properly by allowing substring with some restrictions (e.g., taking into account spaces/underscores in chromosome names). We will work on this during our next release preparation.

Currently you can update either GFF or FASTA to make chromosomes exactly matching. Sorry for the inconvenience!

**Comment by sammyjava** [comment]
Thanks! I think most FASTA routines treat the first space as a separator between the sequence name and a comment, so I'd just go with that "standard." Спасибо!

**Comment by alexeigurevich** [comment]
Fixed in the master branch as you suggested -- just skipping everything after the first whitespace in FASTA entries. Will be available in the next Quast release (5.0.1 or 5.1.0, not clear yet).
Пожалуйста!

**Comment by alexeigurevich** [comment]
Hi @sammyjava, I've just noticed that you are using QUAST for assessing RNA data and a Trinity assembly. You may be interested in [rnaQUAST](http://cab.spbu.ru/software/rnaquast/) which is specifically designed for such evaluations. And probably in [rnaSPAdes](http://cab.spbu.ru/software/rnaspades/) too -- for assembling the data and compare its output with Trinity results.
